.azure-pipelines/client.yml

```diff
@@ -87,6 +87,18 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
+      - powershell: |
+          Invoke-WebRequest -Uri "https://github.com/chidozieononiwu/azure-sdk-for-net/releases/download/verifyAgentOS_v1.1/verify_agent_os.zip" `
+          -OutFile "verify_agent_os.zip" | Wait-Process; Expand-Archive -Path "verify_agent_os.zip" -DestinationPath "./azure-sdk-tools/"
```

Comment:
Is Wait-Process actually needed? I'd expect Invoke-WebRequest would block until it was done.

Reply:
I though so too, but it skips along to try and extract the zip before the download completes.

Reply:
Did you find a reference online somewhere that prompted you to call wait-process? If so could you share the link?